corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -534,6 +544,19 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             });
         },
 
+        updateSearchResults: function () {
+            var self = this;
```

Comment:
no `const` and `let`?

Reply:
ah I was following convention of the rest of the file, but i think this is worth going back to modifying for the whole file if we see fit in a separate PR